add an endpoint to the headling API to update the configuration

zaibon opened the pull request:
fixes #101

zaibon committed:
add an endpoint to the headling API to update the configuration

fixes #101

muhamadazmy reviewed (commented):
I think you have to write the config down on disk, otherwise if minio crashes and restarted by zinit it won't pick up the new config, instead it will use the last configuration it had on disk

zaibon committed:
flist entrypoint: only create config file if it doesn't not exist
already

this avoid to overwrite configs that migh have been updated along the
way in case of a container restart

zaibon committed:
healing API: store config on disk when cfg update is received
